### ISSUE TITLE
Steps from the README don't work

### ISSUE THREAD

**Issue by delneg** [rastreus/FableStarter#1]
After dotnet build:
```
 ~/Documents/FableStarter  main ?1 ▓▒░ dotnet build                                                                                                                              ░▒▓ ✔  19s  12:15:13 
Microsoft (R) Build Engine version 17.0.0+c9eb9dd64 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  Restored /Users/delneg/Documents/FableStarter/FableStarter.fsproj (in 3.01 sec).
/Users/delneg/Documents/FableStarter/src/App.fs(24,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'w-full'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
/Users/delneg/Documents/FableStarter/src/App.fs(25,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'h-screen'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
/Users/delneg/Documents/FableStarter/src/App.fs(26,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'bg-fable-blue-500'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
/Users/delneg/Documents/FableStarter/src/App.fs(27,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'dark:bg-fable-blue-900'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
/Users/delneg/Documents/FableStarter/src/App.fs(28,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'flex'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
/Users/delneg/Documents/FableStarter/src/App.fs(29,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'flex-col'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
/Users/delneg/Documents/FableStarter/src/App.fs(30,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'justify-center'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
/Users/delneg/Documents/FableStarter/src/App.fs(31,16): error FS0039: The type 'CssClasses<./src/index.css,0,node,tailwind-process.js tailwind.config.js>' does not define the field, constructor or member 'items-center'. [/Users/delneg/Documents/FableStarter/FableStarter.fsproj]
```

**Comment by rastreus** [comment]
Sorry about that. 😓 I missed some stuff from Tailwind 3's [Upgrade Guide](https://tailwindcss.com/docs/upgrade-guide). And it seems like [Zanaptak.TypedCssClasses](https://github.com/zanaptak/TypedCssClasses) doesn't like something about my Tailwind 3 configuration.

I removed Zanaptak.TypedCssClasses, updated the dependencies and confirmed that I could work through the steps from the README.

Thanks for letting me know that this was broken. Sorry that it took long to fix.

**Comment by delneg** [comment]
Thank you for your fix, and just on a sidenote - you shouldn't be sorry for 'taking too long' or smth like that. It's opensource, I'm not paying you or anything like that. As an open-source maintainer myself, I found that mindset like that helps a lot.

I've followed steps in the readme one more time (except with npm and not yarn) and it worked perfectly, however It was neede d to delete global.json at first, because I've dotnet 6.0.100 and it has 6.0.101
